sunpy/map/mapbase.py

```diff
@@ -911,10 +911,12 @@ def exposure_time(self):
         """
         Exposure time of the image.
 
-        This is taken from the 'EXPTIME' FITS keyword.
+        This is taken from the 'XPOSURE' keyword or the 'EXPTIME' FITS keyword,
+        in that order.
         """
-        if 'exptime' in self.meta:
-            return self.meta['exptime'] * self.timeunit
+        exptime = self.meta.get('xposure', self.meta.get('exptime', None))
```

Comment:
```suggestion
        exptime = self.meta.get('xposure') or self.meta.get('exptime')
```
?

Reply:
Won't `get`raise a `KeyError` if the key is missing if there isn't a default?

Reply:
I thought the default was None

Reply:
You're right. I've no idea why I thought it raised an error.